collections/_version2-english/9-fork/2-to-dos.md

```diff
@@ -0,0 +1,70 @@
+{% include zoom-image.html src=image_src caption="Figure 1. chain of universes" %}
+
+### Check monthly to see if a fork occurs
+[Forks]({{glossary_fork}}) last for 60 days, so it is strongly recommended that all REP holders "check in" on the project at least once a month to make sure that no fork is underway or they risk losing all of their REP.
+
+The process of "checking in" just means open the [Augur UI]({{ "/" | absolute_url }}/{{page.collection}}/8-augur-ui.html) or ask around (Discord, Twitter, Reddit, etc.) and see if a fork is underway. If no fork is underway then you can go back to passively holding for a month until your next check in without risk of losing your REP.
+
+Useful links for checking if a fork occurs are:
+ - [Augur Discord server](https://discord.gg/guXC9Bd4)  ⭐*recommended*⭐
+ - The [official Twitter account](https://twitter.com/augurproject)
+ - The [official blog](https://augur.net/blog)
+ - The subreddit [/r/Augur](https://www.reddit.com/r/Augur/)
+
+## After starting a fork (60 days)
+REP holders needs to pick a side and migrate their REP to the [child universe]({{glossary_child_universe}}) they believe aligns with reality within 60 days from starting a [fork]({{glossary_fork}}).
+
+After 60 days from starting a fork, [REP]({{glossary_reputation_token}}) in the [parent universe]({{glossary_parent_universe}}) are [locked]({{glossary_locked_universe}}) and can not be migrated to the child universe permanently.
+
+However, the choice should be considered carefully, because migration is one-way; it cannot be reversed. REP cannot be sent from one [sibling universe]({{glossary_sibling_universe}}])to another.
+
+### What if REP holders don't wish to join in the fork?
+They may sell their [REP]({{glossary_reputation_token}}) before the [fork]({{glossary_fork}}) is over, but beware that if many people exercise this strategy the price of REP may drop leading up to the fork and then rebound in the [winning universe]({{glossary_winning_universe}}) after the fork, this could be a costly strategy.
+
+### You receive 40% ROI by joining in the fork?
+Wether you receive 40% [ROI]({{glossary_roi}}) on your migrated [REP]({{glossary_reputation_token}}) by joining in the [fork]({{glossary_fork}} depends on when and where you migrate your REP. If you migrate your REP during the [forking period]({{glossary_forking_period}}) and the [child universe]({{glossary_child_universe}}) which you migrate it becomes [*winning universe*]({{glossary_winning_universe}}) , then you get 40% ROI on your migrated REP. If you migrate after the forking period, you may not get 40% ROI but you can migrate it without risk of migrating to [*losing universe*]({{glossary_losing_universe}}). Because the [final outcome]({{glossary_final_outcome}}) of the [forking market]({{glossary_forking_market}}) is fixed when the forking period ends.
```

Comment:
I'm pretty sure this isn't true.  You only get 40% ROI for disputes.  Migrating your REP doesn't yield any return in v2 IIRC.  Also, you cannot migrate after the forking period at all, so you take a 100% loss in that scenario.  I'm guessing this was copied from v1 where things were different.

Reply:
> You only get 40% ROI for disputes. Migrating your REP doesn't yield any return in v2 IIRC

I thought migrating your REP during the forking period means taking part in the dispute of the forking market. (I used `the forking period` in the sense that the period between the start of a fork and the finalization of the forking market). 

> you cannot migrate after the forking period at all, so you take a 100% loss in that scenario.

Perhaps I don't understand what you're saying again, but I think `the forking period` has two meanings, which are sometimes confusing. I guess you use "the forking period" in the sense of the 60 days window, and I use it in the sense that the period between the start of a fork and the finalization of the forking market.
(We talked about this on Discord (https://discord.com/channels/378030344374583298/617739423920029779/760737699559768065))

I don't want `the forking period` to have multiple meanings, since it confuses. I want it to have only one meaning that the period between the start of a fork and the finalization of the forking market.

Or, if `the forking period` should mean the 60 days window, what should I call the period between the start of a fork and the finalization of the forking market?

> I'm guessing this was copied from v1 where things were different.

No, this was thought up by myself. (Though most of the rest of this part was copied from your comment which is pinned in Discord)

Reply:
> I thought migrating your REP during the forking period means taking part in the dispute of the forking market. (I used the forking period in the sense that the period between the start of a fork and the finalization of the forking market).

No, you only get credit for "participating in a dispute" if you stake REP in dispute crowdsourcers leading up to the fork.

> I don't want the forking period to have multiple meanings, since it confuses. I want it to have only one meaning that the period between the start of a fork and the finalization of the forking market.

For me, the "forking period" is the 60 day window.  Early finalization of the forking market is "just an optimization" and probably worth a note in the document but I'm not sure we need to give a special name to the length of time between fork start and forking market finalization.

If you feel that intermediate period needs a name, I'm not sure what to call it, but I do think we should call fork start to fork end the "forking period".

Reply:
> No, you only get credit for "participating in a dispute" if you stake REP in dispute crowdsourcers leading up to the fork.

Oh, I see. I'm glad I noticed my misunderstanding. I will add this feature to the document. (The features of Augur that I wrote in the document without your confirmation are generally incorrect 😂)

> For me, the "forking period" is the 60 day window. Early finalization of the forking market is "just an optimization" and probably worth a note in the document but I'm not sure we need to give a special name to the length of time between fork start and forking market finalization.

I see. I regard "forking period" as ”60 day window” from now on. Because if you think so, other people (such as devs, and community members) use this word in the same meaning as you think.
But, as a result of this decision, the sentence in the whitepaper I fixed before👇 becomes incorrect... I decided to focus on Augur.Guide, so I do not fix it now. (I might fix it later.)
![20210115_01](https://user-images.githubusercontent.com/38453357/104671902-0cc9fa00-5722-11eb-961f-3659c96159a5.png)

About the name of the intermediate period, I just wanted to know if it had a name. Now I know it is not named, which means it is not necessary I think.

Reply:
I fixed it by PR https://github.com/msagansk/Augur.Guide/pull/42, so I'm closing this conversation.